pkg/infra/log/term/terminal_logger.go

```diff
@@ -45,7 +53,13 @@ func (l terminalLogger) Log(keyvals ...interface{}) error {
 	r := getRecord(keyvals)
 
 	b := &bytes.Buffer{}
+
+	// To make the log output more readable, we make all log levels 5 characters long
 	lvl := strings.ToUpper(r.level.String())
+	if len(lvl) == 4 {
+		lvl += " "
+	}
```

Comment:
```suggestion
lvl := fmt.Sprintf("%-5s", strings.ToUpper(r.level.String()))
```

Same results, but will work with less than four characters as well 🙂

Reply:
more elegant :) thanks @sakjur